imagersite/imager_profile/models.py

```diff
@@ -0,0 +1,92 @@
+"""Define imager profile class and it's methods."""
+
+
+from django.db import models
+from django.db.models.signals import post_save
+from django.contrib.auth.models import User
+from django.dispatch import receiver
+from django.utils.encoding import python_2_unicode_compatible
+
+# Create your models here.
+
+
+class ImagerActiveManager(models.Manager):
+    """Manager for the ImagerProfile Active API."""
+
+    def get_queryset(self):
+        """Return query of active users."""
+        return super(ImagerActiveManager, self).get_queryset().filter(is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """A profile for users to our application."""
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
Great work using the `on_delete` attribute. I would also use the `related_name` attribute to give the associated user model an attribute that matches the profile. Just makes it easier to make the connection between the two models.